tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php

```diff
@@ -118,6 +118,46 @@ public function provideComponentsClientTypeBasedOnSuite(): array
         return (new TestSetupHelper())->provideComponentsClientTypeBasedOnSuite($this);
     }
 
+    /**
+     * @dataProvider provideComponentsClientTypeBasedOnSuite
+     */
+    public function testIfTryRegisterNonExistingListingItFailWithUserErr(string $devBranchType, string $userRole): void
```

Comment:
```suggestion
    public function testRegisterNonExistingListingFailsWithUserError(string $devBranchType, string $userRole): void
```
```suggestion
    public function testRegisterNonExistingListingFailsWithUserErr(string $devBranchType, string $userRole): void
```

Reply:
jen at ten nazev neni tak dlouhy a dava vetsi smysl.

Reply:
Jo jasne dik, takto je to lepsie :)